packages/network/samba/scripts/samba-autoshare

```diff
@@ -21,6 +21,8 @@ if [ -f /storage/.cache/services/samba.conf ]; then
   . /storage/.cache/services/samba.conf
 
   if [ "$SAMBA_AUTOSHARE" == "true" ] ; then
-    systemctl restart smbd.service
```

Comment:
if possible, ```systemctl reload smbd.service``` should be used and re-runing samba-config and smbd-config should be moved to additional ```ExecReload=```s in smbd.service 
```
[Service]
Type=forking
...
ExecReload=/usr/lib/samba/smbd-config
ExecReload=/usr/bin/systemctl restart samba-config.service
ExecReload=/bin/kill -HUP $MAINPID
...
```
sorta

Reply:
This doesn't work as intended. Udevil mounts drives before samba is started, and a service can't be reloaded until it's active:

```Aug 25 14:06:03 NUC systemd[1]: smbd.service: Unit cannot be reloaded because it is inactive.
Aug 25 14:06:03 NUC samba-autoshare[412]: smbd.service is not active, cannot reload.
Aug 25 14:06:03 NUC systemd[1]: smbd.service: Unit cannot be reloaded because it is inactive.
Aug 25 14:06:03 NUC samba-autoshare[410]: smbd.service is not active, cannot reload.
Aug 25 14:06:03 NUC systemd[1]: smbd.service: Unit cannot be reloaded because it is inactive.
Aug 25 14:06:03 NUC samba-autoshare[662]: smbd.service is not active, cannot reload.
```

It's trying to reload three times, once for System and Flash, then once for my external drive. As the reload doesn't trigger, the external drive isn't shared once Samba starts. If I trigger a reload after, it works as intended.

Not sure if it's possible/feasible to get udev/udevil to start after Samba. The PR as is works satisfactory.

Reply:
> It's trying to reload three times, once for System and Flash

Is your tree up to date?  `udevil-mount` should already ignore `/system` and `/flash` (and any other already mounted partition), see after https://github.com/LibreELEC/LibreELEC.tv/pull/1800

> The PR as is works satisfactory.

Yes, the addition of `pkill -HUP smbd` has addressed my original concerns - thanks (and @stefansaraev).

Reply:
Ah, hadn't seen that change. I was testing this on 8.1 actually.